dev/run_docker_compose.sh

```diff
@@ -35,6 +35,4 @@ if [ ! -d parquet-cpp ]; then
     exit 1
 fi
 
-GID=$(id -g ${USERNAME})
-docker-compose -f arrow/dev/docker-compose.yml run \
-               --rm -u "${UID}:${GID}" "${1}"
+docker-compose -f arrow/dev/docker-compose.yml run --rm "${1}"
```

Comment:
Would you mind changing the arguments in the docker-compose command to pass all and not just the first?  So change `"${1}"` to `"${@}"`.  This will allow to run it like the following to map the local Maven repo and avoid downloading all the artifacts to the image if they have already been downloaded locally:
```
run_docker_compose.sh -v $HOME/.m2/:/root/.m2 spark_integration
```

Reply:
Sure!